packages/bc-wallet-traction-adapter/Dockerfile

```diff
@@ -15,12 +15,10 @@ rm -rf /var/lib/apt/lists/*
 
 WORKDIR /build
 COPY package.json pnpm-lock.yaml pnpm-workspace.yaml turbo.json tsconfig.json tsconfig.build.json tsconfig.eslint.json ./
-COPY packages/bc-wallet-traction-adapter packages/bc-wallet-traction-adapter/
-COPY packages/tsconfig* packages/
+COPY packages packages
```

Comment:
Can't we use Docker multi-stage build to remove the unnecessary packages?

Reply:
It is a Docker multi-stage build, only the /deploy dir is going the the second phase.
The build phase just needs to build ./packages to be able to resolve current packages and packages added in the future. Updating all Dockerfiles every 5 minutes we add a new package does not seem practical and unused packages do not end up in the final build.